frontendVfsUtils/src/main/java/com/virtuslab/gitmachete/frontend/file/GitVfsUtils.java

```diff
@@ -1,4 +1,4 @@
-package com.virtuslab.gitmachete.frontend.vfs.utils;
+package com.virtuslab.gitmachete.frontend.file;
```

Comment:
Hmm I'd keep a different package here... since IIUC `file` package in `frontendFile` refers to .git/machete file, while here it refers to VFS files in general :thinking:

Reply:
`vfs` or `vfsutils` (single word) would be also ok, up to you

Reply:
(unless we might have `vfs.utils` and `vfs.somethingelse`)